### ISSUE TITLE
Name of participant is nil

### ISSUE THREAD

**Issue by Deviz** [layerhq/Layer-Parse-iOS-Example#24]
To generate issue do following:

1) Run app
2) Select conversation
3) View that the name of participant is shown in AddressBar
4) Go back to ConversationListView
5) Toggle the home button (while app is running)
6) Open app 
7) Select conversation (the participant name is no longer shown)

If you go back and tap a conversation again then name of participant is shown again.

I think the problem is in ConversationViewController.m
Maybe something to do with the comment you made in the file:
// TODO: Need a good way to refresh all the messages for the refreshed participants...

Looking forward to your answer :)

**Comment by glaksmono** [comment]
I'm having this issue as well, anyone got ideas to solve this? @Deviz

**Comment by glaksmono** [comment]
I figured that I think these 2 functions needs to be called first, then this problem wouldn't be happening:

```
    func conversationListViewController(conversationListViewController: ATLConversationListViewController!, avatarItemForConversation conversation: LYRConversation!) -> ATLAvatarItem!
    
    func conversationListViewController(conversationListViewController: ATLConversationListViewController, titleForConversation conversation: LYRConversation) -> String
```

**Comment by glaksmono** [comment]
Or may be loading `ConversationListViewController` in the background?

**Comment by blakewatters** [comment]
Hi @glaksmono -

I think that the problem is that the query controller isn't being set up until the view will appear. I think you probably want to try moving:

```objc
    if (!self.queryController) {
        [self setupConversationDataSource];
    }
```
from `viewWillAppear:` into `viewDidLoad:` and that will avoid the crash.

**Comment by Deviz** [comment]
It's not working. Any solutions yet? @glaksmono @blakewatters

**Comment by glaksmono** [comment]
@Deviz nope, they just released a new version of the SDK though, have you tried using that? I tried it, but it introduces new issues, requested @blakewatters to open sourced the example using the latest SDK

**Comment by Deviz** [comment]
@glaksmono No, I haven't updated. I was afraid that the new SDK will break the app. What kind of issues does it introduce? And is it the Layer SDK or is it the Atlas UI?

**Comment by Deviz** [comment]
I have updated to the newest LayerKit version 0.19.3 and updated to Atlas 1.0.19, but this issue is still around - any ideas? @blakewatters @glaksmono
